SwatDB/SwatDBDataObject.php

```diff
@@ -687,6 +710,43 @@ protected function getDeprecatedProperty($key)
 	}
 
 	// }}}
+	// {{{ protected function getProtectedPropertyList()
+
+	/**
+	 * Gets a list of all protected properties of this data-object
+	 *
+	 * Protected properties should correspond directly to database fields.
```

Comment:
Array format needs to be defined. Are the values arrays with 'get' and 'set' keys?